src/wasm.rs

```diff
@@ -10,20 +10,12 @@ use std::task::{Context, Poll};
 
 /// WebAssembly HTTP Client.
 #[derive(Debug)]
-pub struct WasmClient {
-    _priv: (),
-}
```

Comment:
We need to keep this to ensure people can't create clients by doing:

```rust
let client = WasmClient {};
```

It's a way of forcing people to use `::new`, giving us the option to always change the internal fields later